scripts_bash/1_data_creation/organize_from_tractoflow.sh

```diff
@@ -0,0 +1,84 @@
+# checked!
```

Comment:
I think the file should be renamed to `organize_from_tractoflow_template.sh`, to make it clear that it should be modified before running

Reply:
Ok. So far, the other ones were named example_... so I will do that here.